test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -763,7 +763,7 @@ TEST_VM(os, find_mapping_simple) {
       ASSERT_EQ(mapping_info.base, p);
       ASSERT_EQ(mapping_info.regions, 1);
       ASSERT_EQ(mapping_info.size, total_range_len);
-      ASSERT_EQ(mapping_info.committed_size, 0);
+      ASSERT_EQ(mapping_info.committed_size, size_t { 0 });
```

Comment:
```suggestion
      ASSERT_EQ(mapping_info.committed_size, size_t{0});
```
We don't usually insert extra spaces. Was there a special reason?

Reply:
No, there wasn't really a reason other than the formatting looked neater. But since the style is to usually not leave spaces, I'll commit the suggestion and fix the other areas too